shared/test/test_assets.rb

```diff
@@ -154,6 +154,20 @@ def test_assets_quota
     end
   end
 
+  def test_asset_last_modified
+    channel = create_channel(@channels)
+
+    start_time = DateTime.now
+    put @assets, channel, 'test.png', 'not a real image', 'image/png'
+
+    get @assets, channel, 'test.png', '', 'HTTP_IF_MODIFIED_SINCE' => start_time.httpdate
```

Comment:
I want to verify that we return `200 OK` when `If-Modified-Since` refers to an old revision.  Setting this to a date before the asset is created seems safe?